### PR TITLE
Sort imports

### PR DESCRIPTION
**Pull request by felixhasse**
Update the prettier config to sort imports. For that to work, https://github.com/trivago/prettier-plugin-sort-imports needs to be installed as a devDependency in the parent project.

Imports are sorted in the following order:
1.  React imports
2. Next.js
3. Mayd
4. Imports from installed packages
5. Relative imports

Within these categories, imports are sorted alphabetically

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -5,5 +5,15 @@
...
+        "^react(.*)$",
+        "^next(.*)$",
+        "^@mayd/(.*)$",
+        "^[^.@].*",
+        "^@(.*)$",
+        "^[./]"
+    ],
+    "importOrderSeparation": true,
+    "importOrderSortSpecifiers": true
+}
```
Comment (mangoischke) [inline .prettierrc.json]:
```suggestion
}

```

File: README.md [hunk 2]
```diff
@@ -24,3 +24,9 @@ You will have to install prettier as a dev dependency, too
 ```
 npm install --save-dev prettier
```
Comment (mangoischke) [inline README.md]:
```suggestion
npm install --save-dev prettier @trivago/prettier-plugin-sort-imports
```

File: README.md [hunk 3]
```diff
@@ -24,3 +24,9 @@ You will have to install prettier as a dev dependency, too
...
 ```
+Enable sorting imports
+------------
+To enable automatically sorting imports, you additionally need to install the following package:
+```
+npm install --save-dev @trivago/prettier-plugin-sort-imports
+```
```
Comment (mangoischke) [inline README.md]: 👋

**Review by mangoischke — CHANGES_REQUESTED** [review]

File: .prettierrc.json [hunk 4]
```diff
@@ -5,5 +5,15 @@
...
+    "arrowParens": "avoid",
+    "importOrder": [
+        "^react(.*)$",
+        "^next(.*)$",
+        "^@mayd/(.*)$",
+        "^[^.@].*",
+        "^@(.*)$",
+        "^[./]"
+    ],
+    "importOrderSeparation": true,
```
Comment (timfuhrmann) [inline .prettierrc.json]: As discussed:
```suggestion
    "importOrderSeparation": false,
```

File: .prettierrc.json [hunk 5]
```diff
@@ -5,5 +5,15 @@
     "printWidth": 100,
     "singleQuote": false,
     "bracketSameLine": true,
-    "arrowParens": "avoid"
+    "arrowParens": "avoid",
+    "importOrder": [
+        "^react(.*)$",
+        "^next(.*)$",
+        "^@mayd/(.*)$",
```
Comment (timfuhrmann) [inline .prettierrc.json]: And I guess we should escape `/`, shouldn't we?

```suggestion
        "^next(.*)$",
        "^@fr(e|a)ym\/(.*)$",
        "^@mayd\/(.*)$",
```

**Review by timfuhrmann — CHANGES_REQUESTED** [review]

**Review by timfuhrmann — APPROVED** [review]

**Pull request merged**